tests/queries/0_stateless/01338_sha256_fixedstring.sql

```diff
@@ -0,0 +1,23 @@
+SELECT SHA256('');
+SELECT SHA256('abc');
+
+DROP TABLE IF EXISTS defaults;
+CREATE TABLE defaults
+(
+    s FixedString(20)
+)ENGINE = Memory();
+
+INSERT INTO defaults SELECT randomFixedString(20);
+INSERT INTO defaults SELECT randomFixedString(20);
+INSERT INTO defaults SELECT randomFixedString(20);
+INSERT INTO defaults SELECT randomFixedString(20);
+INSERT INTO defaults SELECT randomFixedString(20);
+INSERT INTO defaults SELECT randomFixedString(20);
+INSERT INTO defaults SELECT randomFixedString(20);
+INSERT INTO defaults SELECT randomFixedString(20);
+INSERT INTO defaults SELECT randomFixedString(20);
+INSERT INTO defaults SELECT randomFixedString(20);
```

Comment:
```suggestion
INSERT INTO defaults SELECT s
FROM generateRandom('s FixedString(20)', 1, 1, 1)
LIMIT 20
```

Reply:
randomFixedString is not stable between different calls. You can use predefined seed in `generateRandom` instead.

Reply:
ok, i will change it.